### PR TITLE
Fail fast on language keywords in method names

### PR DESCRIPTION
**Pull request by andreaTP**
References #239

Tested with scripted tests since I haven't found a more suitable place.
This can be a first proposal that provides the user a better error message (it's going to fail the build anyhow).
We can, later on, decide to backtick those keywords in Scala.

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

**Review by raboof — APPROVED** [review]
I agree, nice improvement already!

**Comment by raboof** [comment]
> We can, later on, decide to backtick those keywords in Scala

Oh, cool, this is already in progress in #1236

**Comment by andreaTP** [comment]
Minor issue:
Travis keeps failing because my fork master branch is pretty old and sbt get a `previousStableVersion` that doesn't exists.
Ref.
https://github.com/akka/akka-grpc/blob/master/build.sbt#L54

**Comment by andreaTP** [comment]
Is Jenkins stuck?

**Comment by raboof** [comment]
OK TO TEST

File: codegen/src/main/scala/akka/grpc/gen/scaladsl/Method.scala [hunk 1]
```diff
@@ -18,6 +18,10 @@ case class Method(
     comment: Option[String] = None)(implicit ops: DescriptorImplicits) {
   import Method._
 
+  require(
+    !ReservedWords.contains(name),
```
Comment (sebastian-alfers) [inline codegen/src/main/scala/akka/grpc/gen/scaladsl/Method.scala:22]: All scala related fail-fast logic should be removed right?

Comment (andreaTP) [reply codegen/src/main/scala/akka/grpc/gen/scaladsl/Method.scala:22]: yes!

Comment (andreaTP) [reply codegen/src/main/scala/akka/grpc/gen/scaladsl/Method.scala:22]: I'll rebase after #1236 get merged.

**Comment by andreaTP** [comment]
rebased on latest master.

**Review by raboof — APPROVED** [review]
Thanks for the rebase!

**Pull request merged**